xapian-applications/omega/omega.cc

```diff
@@ -360,6 +372,13 @@ try {
     for (auto i = begin; i != end; ++i) {
 	const string & v = i->second;
 	if (!v.empty()) {
+	    if (i->first[CONST_STRLEN("START.")] < '0' ||
+		i->first[CONST_STRLEN("START.")] > '9' ||
+		i->first[CONST_STRLEN("START.")] == '-') {
+		cerr << "Format entered is not valid\n"
+		    << "Valid formats are YYYY,YYYYMM,"
+		    << "YYYYMMDD or YYYYMMDDHHMM";
```

Comment:
So we aren't really enforcing that the format is valid, e.g. `12345` is quietly accepted but doesn't match any of those.

Reply:
thanks i will fix that.